doc/invalidation-handlers.rst

```diff
@@ -0,0 +1,71 @@
+Extra Invalidation Handlers
+===========================
+
+This library provides decorators that build on top of the CacheInvalidator to
+simplify common operations.
+
+.. _tags:
+
+Tag Handler
+-----------
+
+The tag handler helps you to invalidate all cache entries that where marked
+with a specified tag. It works only with a CacheInvalidator that supports
```

Comment:
Double backticks for CacheInvalidator (or does this repo use different standards?)